tox.ini

```diff
@@ -15,11 +15,7 @@ deps =
 passenv = ARCHFLAGS LDFLAGS CFLAGS INCLUDE LIB LD_LIBRARY_PATH USERNAME PYTHONIOENCODING OPENSSL_FORCE_FIPS_MODE
 commands =
     pip list
-    # We use parallel mode and then combine here so that coverage.py will take
-    # the paths like .tox/py38/lib/python3.8/site-packages/cryptography/__init__.py
-    # and collapse them into src/cryptography/__init__.py.
-    coverage run --parallel-mode -m pytest --capture=no --strict --durations=10 {posargs}
-    coverage combine
+    pytest -n 2 --cov=src/cryptography --cov=tests --capture=no --strict --durations=10 {posargs}
```

Comment:
Is there a `-n auto` or something?

Reply:
There is, but I'm reluctant to use it because it will spawn more processes than are useful quite easily (e.g. on my laptop it spawns 12, which is slower than just doing 4 on this 6 core/12 thread machine).

Reply:
That's... interesting... do we have any idea why it gets slower beyond 4 cores?

Reply:
As best I can tell it's a function of the cost of collection and merging after test completion. I can switch to auto though because it doesn't really matter for our CI (and I don't run tox locally).

Reply:
SGTM